lib/scala/logging-config/src/main/java/org/enso/logger/config/LoggersLevels.java

```diff
@@ -55,6 +55,15 @@ public boolean isEmpty() {
     return loggers.isEmpty();
   }
 
+  /**
+   * Checks if any of the custom levels refer to Enso's loggers.
+   *
+   * @return true if any of the custom log levels are for `org.enso.*` class
+   */
+  public boolean hasEnsoLoggers() {
```

Comment:
Why check for `hasEnsoLoggers`? Don't we want to enabled for example also JavaScript or Python loggers dynamically?

Reply:
I didn't think of that case, TBH. Can you give me an example?

Reply:
I haven't found any use of `TruffleLogger` in JavaScript. GraalPython seems to be using [TruffleLogger somehow](https://github.com/oracle/graalpython/blob/a9b6cd7edc6e25f9b1d4011316e2d153ee88150f/graalpython/com.oracle.graal.python/src/com/oracle/graal/python/PythonLanguage.java#L291).

Reply:
`regex` uses TruffleLoggers for sure. And they are pretty verbose.

Reply:
I am not sure if I fully understand this discussion. But if this change means that we cannot suppress, for example, `org.truffle.r.*.logger` or `org.graalpython*.logger`, then that is unfortunate.